src/Core/Framework/Plugin/PluginManagementService.php

```diff
@@ -60,12 +55,14 @@ public function extractPluginZip(string $file, bool $delete = true, ?string $sto
 
         if ($storeType) {
             $this->pluginExtractor->extract($archive, $delete, $storeType);
+            $this->cacheClearer->clearContainerCache();
```

Comment:
Here should be an if when the storeType is an APP don't clear the container cache